### PR TITLE
Unify project name as "Liquid Prompt" in docs (spelled as two words)

### PR DESCRIPTION
**Pull request by ottok**
On the front page README.md the project name is spelled "Liquid Prompt" as two words. This PR suggests to use that spelling across all project documentation.

### CODE REVIEW

**Comment by Rycieos** [comment]
In my opinion, I think it makes sense as one word. The project ID is also one word; if it was two words, I would think it would be `liquid-prompt`. I would rather see any instances of "Liquid prompt" changed to "Liquidprompt" than this.

But @nojhan named it originally, so what would you like to see?

**Comment by augmentedfourth** [comment]
Also note that the package name in homebrew (and I think other distribution mechanisms) is a single word, un-capitalized... just like the file name of the script itself. I'll defer to @nojhan as well, but as someone who's used this tool for over a decade "Liquid Prompt" just looks jarring.

**Comment by nojhan** [comment]
That's a thought question. Do someone have access to some SEO tools to check which name leads to the largest search engine share (out of existing hits)?

**Comment by Rycieos** [comment]
> Do someone have access to some SEO tools to check which name leads to the largest search engine share

I'm no SEO expert, but some quick digging leads to the following:
1. "Liquid prompt" has more than 10 times the "hits" than "Liquidprompt" does, but that is not a fair comparison. The space means it is two separate search terms, and you get a ton of false positive hits.
2. "Liquid prompt" hits mostly on blog posts about the project (so many blog posts, wow).
3. "Liquidprompt" hits mostly on documentation and packaging indexes.
4. "Liquidprompt" hits on pages with only the string "Liquid prompt" in the page.

Based on this, I think:
* Because of 4, searching with the string "liquidprompt" will hit both forms, while "liquid prompt" won't always.
* Documentation is probably the first thing we want users to find.
* Because of 1, we don't want users to find unrelated pages.

I think that "Liquidprompt" is better for SEO.

**Comment by nojhan** [comment]
Seems like a good point to me :)

**Comment by ottok** [comment]
I'll wait for a week for further feedback and if this is the decision, I'll revert the PR to do the change for all current Liquid Prompt into Liquidprompt.

**Comment by dolmen** [comment]
As someone who has contributed to this mess, my preference was Liquid Prompt as the project name and `liquidprompt` as the command to invoke it.

But as a former maintainer, consider my vote as neutral if votes are involved.

**Comment by ottok** [comment]
When looking at pages that Google finds on "liquidprompt" they are all pages that reference the package name, not the project name. As soon as you start to get results about blog posts that decribe the project, everybody seem to write about "Liquid Prompt" – two separate words, and both capitalized. I agree with that and I think it is also better English (though I am not a native speaker). One of the search results with "liquidprompt" even yields "Liquid Prompt", because that blog post is so popular.

Thus, I leave my PR leaning on "Liquid Prompt".


image

vs

image

**Comment by ottok** [comment]
I also checked [Google Trends](https://trends.google.com/trends/explore?date=all&q=liquid%20prompt,liquidprompt). The name "Liquid Prompt" is more popular than spelled together, but the search volume is so low that the difference is not significant.

image

**Comment by Rycieos** [comment]
Sorry for the radio silence on this. I'm not sure what the path forward should be, since we have a few on each side of the camp, and @nojhan seems to be fine either way. Ideally we would get some other opinions on this, but I realize that leaving this sitting open forever is not ideal either.

**Comment by ottok** [comment]
Don't worry, leaving the issue open is fine to me, no hurry here :)

In this case taking some time to collect feedback and think about it is a
good thing.

>

**Comment by nojhan** [comment]
Looking at the search engine results, I think "Liquid Prompt" would better match what the actual users are using to refer to the project.

This is a key point in my opinion, because we are facing competitors that have less interesting features and design, and still surprisingly seem to be more popular. We definitely need to reach the users *in their own words*.

Also, it emphasizes one of the key feature of the design, which is to actually be *liquid*. It is probably worth it to use the adjective as a keyword, and I think having it separated helps. One could even use "*the* Liquid Prompt", so as to emphasize even more.

It doesn't seem like a big deal to have the command/repository keeping the one-word version, I think most of people would understand, especially if the official website use the two-words version.

File: docs/theme/included/powerline.rst
Comment (Rycieos) [inline docs/theme/included/powerline.rst:295]: Line needs expanding to match header.

File: docs/theme/included/powerline.rst
Comment (Rycieos) [inline docs/theme/included/powerline.rst:66]: Line needs expanding to match header.

File: docs/theme/included/alternate_vcs.rst
Comment (Rycieos) [inline docs/theme/included/alternate_vcs.rst:53]: Line needs expanding to match header.

File: docs/index.rst
Comment (Rycieos) [inline docs/index.rst:7]: Line needs expanding to match header.

File: docs/config.rst
Comment (Rycieos) [inline docs/config.rst:128]: This isn't a explicit name reference, but a directory path. Probably should leave this one as it was, even though it is only an example.

**Review by Rycieos — CHANGES_REQUESTED** [review]
Ok, sounds like a close enough consensus then. I agree, let's unify the name.

@ottok, if you could fix these small issues I've pointed out, and maybe squash and rebase, I'll merge this in.

**Comment by ottok** [comment]
Sure, I can refresh to use "Liquid Prompt" everywhere as agreed today.

**Comment by ottok** [comment]
Fixed all items commented and rebased on latest master.

**Review by Rycieos — APPROVED** [review]
Thanks @ottok!

**Pull request merged**
